commit 38cff47157a8f073de2a7e81fd00557a8a150222
Author: Jeroen Demeyer <jdemeyer@cage.ugent.be>
Date:   Wed Aug 17 13:44:43 2016 +0200

    Replace "int(val) == 0" by "not val" for gdb.Value instances

    See https://github.com/cython/cython/pull/1439

diff --git a/Cython/Debugger/libpython.py b/Cython/Debugger/libpython.py
index b43b83c..2158c1f 100644
--- a/Cython/Debugger/libpython.py
+++ b/Cython/Debugger/libpython.py
@@ -286,7 +286,7 @@ class PyObjectPtr(object):
         return PyTypeObjectPtr(self.field('ob_type'))
 
     def is_null(self):
-        return 0 == int(self._gdbval)
+        return not self._gdbval
 
     def is_optimized_out(self):
         '''
@@ -989,7 +989,7 @@ class PyFrameObjectPtr(PyObjectPtr):
         if self.is_optimized_out():
             return None
         f_trace = self.field('f_trace')
-        if int(f_trace) != 0:
+        if f_trace:
             # we have a non-NULL f_trace:
             return self.f_lineno
         else:
